tests/cypress/integration/actions_tasks3/case_109_dummy_cloud_storage.js

```diff
@@ -0,0 +1,196 @@
+        status, id, description, provider,
+    }) {
+        cy.get('.cvat-cloud-storage-item').should('have.length', 1);
+        cy.get('.cvat-cloud-storage-item')
+            .should('contain', `Status: ${status}`);
+        if (description) {
+            cy.get('.cvat-cloud-storage-item')
+                .should('contain', `#${id}: ${description}`);
+        }
+        if (provider) {
+            cy.get('.cvat-cloud-storage-item')
+                .should('contain', `Provider: ${provider}`);
+        }
+        if (status !== 'Error') {
+            cy.get('.cvat-cloud-storage-item-preview').should('exist');
+        }
+    }
+
+    function testCSSetStatusPreview(id, status, image) {
+        cy.intercept('GET', `api/v1/cloudstorages/${id}/status`, status).as('csStatus');
+        cy.intercept(
+            'GET',
+            `api/v1/cloudstorages/${id}/preview`,
+            { fixture: `${imageFolder}/${image}` },
+        ).as('csPreview');
+
+        cy.contains('.cvat-header-button', 'Models').should('be.visible').click();
+        cy.contains('.cvat-header-button', 'Cloud Storages').should('be.visible').click();
+
+        cy.wait('@csStatus').its('response.statusCode').should('eq', 200);
+        cy.wait('@csPreview').its('response.statusCode').should('eq', 200);
+    }
+
+    function testGoToCSUpdatePage() {
+        cy.get('.cvat-cloud-storage-item-menu-button').trigger('mousemove').trigger('mouseover');
+        cy.get('.ant-dropdown')
+            .not('.ant-dropdown-hidden')
+            .within(() => {
+                cy.contains('[role="menuitem"]', 'Update').click();
+            });
+
+        cy.get('.cvat-notification-update-info-cloud-storage').should('exist');
+
+        cy.get('.cvat-cloud-storage-form').should('be.visible');
+    }
+
+    beforeEach(() => {
+        cy.visit('auth/login');
+        cy.login();
+    });
+
+    afterEach(() => {
+        cy.logout();
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Create dummy Google cloud storage and check fields.', () => {
+            testCreateDummyStorage(cloudStoragesDummyDataGoogleStorage);
+            testCheckAndCloseNotification();
+            testCSValues({
+                status: 'Error', id: 3, description: 'Demo GCS', provider: 'GOOGLE_CLOUD_STORAGE',
+            });
+            testCSSetStatusPreview(3, 'NOT_FOUND', 'preview_GOOGLE_CLOUD_STORAGE.png');
+            testCSValues({
+                status: 'NOT_FOUND',
+            });
+            testGoToCSUpdatePage();
+
+            cy.get('#display_name')
+                .should('be.visible')
+                .and('have.value', 'Demo GCS');
+            cy.get('#description')
+                .should('be.visible')
+                .and('have.value', 'It is first google cloud storage');
+            cy.get('.cvat-cloud-storage-select-provider')
+                .should('be.visible')
+                .and('have.text', 'Google Cloud Storage');
```

Comment:
```suggestion
                .should('be.visible')
                .should('be.disabled')
                .and('have.text', 'Google Cloud Storage');
```

Reply:
That's how the error will be. Since id ```provider_type``` is disabled and visibility and text can be checked from class ```cvat-cloud-storage-select-provider```